---
layout: post
title: A cluster-based cell-type annotation method
subtitle: iScience, 2020
thumbnail-img: /assets/img/sccatch_j.png
tags: [Paper]
---

Recent advancements in single-cell RNA sequencing (scRNA-seq) have facilitated the classification of thousands of cells through transcriptome profiling, wherein accurate cell type identification is critical for mechanistic studies. In most current analysis protocols, cell type-based cluster annotation is manually performed and heavily relies on prior knowledge, resulting in poor replicability of cell type annotation. This study aimed to introduce a single-cell Cluster-based Automatic Annotation Toolkit for Cellular Heterogeneity (scCATCH, https://github.com/ZJUFanLab/scCATCH). Using three benchmark datasets, the feasibility of evidence-based scoring and tissue-specific cellular annotation strategies were demonstrated by high concordance among cell types, and scCATCH outperformed Seurat, a popular method for marker genes identification, and cell-based annotation methods. Furthermore, scCATCH accurately annotated 67%-100% (average, 83%) clusters in six published scRNA-seq datasets originating from various tissues. The present results show that scCATCH accurately revealed cell identities with high reproducibility, thus potentially providing insights into mechanisms underlying disease pathogenesis and progression.

<hr style="max-width:100%;height:1px;background:#eaeaea;border:none;">

<h3><a href="https://doi.org/10.1016/j.isci.2020.100882">scCATCH: Automatic Annotation on Cell Types of Clusters from Single-Cell RNA Sequencing Data</a></h3>
<div style="text-align: center;padding-top: 20px;padding-bottom: 20px;">
  <a href="https://doi.org/10.1016/j.isci.2020.100882">
  <img src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/22/4/10.1093_bib_bbaa269/1/bbaa269f1.jpeg?Expires=1746280930&Signature=AlMn5EzQzR5N-E7D1BzFIRCqlkJfasYFl3-ZG2QK3tZH5VlMYp3sqyYq7J3BNvlGbZWnjcjdOdTd6DS6B9RYFMWOf4fi9nx-P8T0-TIeQduOloMIlzTMfZT2yxm2g9p1mZ04aZLA~QeI1rUQI0UdAKL9wcwEybr9dRYMDMPqcSFM5P5lKqGz5tOICgDILcHUoCFzw20Wi-OxLqq04TEkyUxx1O8hQ~TP9gv3rCf7VaYueiKHyzV3h4P9iNLIxAWGl7uc1ijTIkD6jdpynWm9PKVpg1hUEsc9gz-EN4gbYC2yxmWTXyvxIuNV7GvyoZtQFnOwoQK40pYKdes32zkeFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="scDeepSort" style="width: 90%; height: auto;transition: transform 0.3s ease;" onmouseover="this.style.transform='scale(1.05)'" onmouseout="this.style.transform='scale(1)'" />
  </a>
</div>

<hr style="max-width:100%;height:1px;background:#eaeaea;border:none;">

<h3><a href="https://xomics.com.cn/celltalkdb/">URL</a></h3>
<div>
<a href="https://xomics.com.cn/celltalkdb/">https://xomics.com.cn/celltalkdb/</a>
</div>

<h3><a href="https://doi.org/10.1016/j.isci.2020.100882">Publication</a></h3>
<div>
<a href="https://doi.org/10.1016/j.isci.2020.100882">https://doi.org/10.1016/j.isci.2020.100882</a>
</div>
