---
layout: post
title: A drug-responsive cell type inference method
subtitle: Cell Reports Medicine, 2024
thumbnail-img: /assets/img/scrank_j.png
tags: [Paper]
---

Cells respond divergently to drugs due to the heterogeneity among cell populations. Thus, it is crucial to identify drug-responsive cell populations in order to accurately elucidate the mechanism of drug action, which is still a great challenge. Here, we address this problem with scRank, which employs a target-perturbed gene regulatory network to rank drug-responsive cell populations via in silico drug perturbations using untreated single-cell transcriptomic data. We benchmark scRank on simulated and real datasets, which shows the superior performance of scRank over existing methods. When applied to medulloblastoma and major depressive disorder datasets, scRank identifies drug-responsive cell types that are consistent with the literature. Moreover, scRank accurately uncovers the macrophage subpopulation responsive to tanshinone IIA and its potential targets in myocardial infarction, with experimental validation. In conclusion, scRank enables the inference of drug-responsive cell types using untreated single-cell data, thus providing insights into the cellular-level impacts of therapeutic interventions.

### Overview
<div style="text-align: center;">
  <img src="https://github.com/user-attachments/assets/682b5d00-520b-4238-b495-a0d303f4273d" alt="scRank" style="width: 80%;" />
</div>

### URL
<div>
[https://github.com/ZJUFanLab/scRank](https://github.com/ZJUFanLab/scRank)
</div>

### Publication
<div>
[https://doi.org/10.1016/j.xcrm.2024.101568](https://doi.org/10.1016/j.xcrm.2024.101568)
</div>